### PR TITLE
Tweaks Turbo and Adds Antivenom

### PR DESCRIPTION
**Pull request by SwiGs1**
## Description
Adds antivenom as a tribal and definitive option for dealing with toxin damage instead of relying on cave fungus and barrel cactus, and tweaks turbo so it's not just useless and give back the fast effective, it's not as fast as it used to be but it's something I guess. As well as gives Radscorpions their own venom reagent for something I will work on later

## Motivation and Context
Antivenom gives the wasteland and mostly tribals (Legion and stuff) some options with dealing with toxin damage instead of having to rely on cave fungus or barrel cactus or use BAD CHEMS. It's easy to craft and I hope the sprite is okay.  It's been a while since I've coded so I wanted to try to add something simple and give some options in regards to medicine. As for turbo, I've given back the effect that made you go SUPER DUPER FAST which was OP. But instead just slowed it a down a bit so you go FASTA! but not too FAST. Saves you from having to paint things red. Turbo was OP but it doesn't deserve to get nerfed to shit, denying slowdown was med-x's job, but I was unsure about removing slowdown so, well see how it goes I guess, in short turbo's gotta go back to being speed drug. 

## How Has This Been Tested?
Tested Locally in the SipsCo-Swigs Industries LLC Laboratory. 

## Screenshots (if appropriate):
Screenshot (719)
Screenshot (720)
test2
test1
Screenshot (721)

## Changelog (necessary)
:cl:
add: antivenom bottle, reagent and crafting recipe
tweak: gives Turbo back its speed effect but dulled down.
edit: gives radscorpions their own venom reagent
/:cl:

### CODE REVIEW

**Comment by waterwater1212** [comment]
This is a Liquid Gang Production.

Sponsored by Sips Co. Enterprises

Funded by The S Foundation

**Comment by FantasticFwoosh** [comment]
:-1: will be used to rush green goo covered ruins and dungeons with no health consequence.

Grind up barrel cactus in a bone pestle & mortar, or a wood barrel, scoop out 10u exactly and drink the prescribed amount of medication and you will be fine.

**Comment by SwiGs1** [comment]
Probably could remove slowdown trait instead and just leave the gofast trait instead of the goreallyfast trait it had before, I personally liked how turbo used to work before and nerf it got wasn't needed imo. As I said before it just gives some extra and easier options for dealing with toxin damage and whatnot, instead of having to go out doing pretty meagre things in the instance you get stingy stung by Cazadores, Idk. Testing waters here right now, We'll see what happens i guess.

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -1836,6 +1836,14 @@
 	time = 80
 	category = CAT_MEDICAL
 
+	/datum/crafting_recipe/antivenom
+	name = "Antivenom"
+	result = /obj/item/reagent_containers/glass/bottle/antivenom
```
Comment (Afya212) [inline code/modules/crafting/recipes.dm:1841]: I'm not a fan of glass reagent container magically appearing out of no where. I'd like it if it required a sheet or two of glass, or `/obj/item/reagent_containers/glass/beaker`, which is craft-able.

Comment (SwiGs1) [reply code/modules/crafting/recipes.dm:1841]: All done.

**Comment by imw-Oculus** [comment]
Turbo buff looks great to me, used it yesterday and was taken aback by how little the speed buff is. As for the anti-venom, eh. Xander already contains anti-toxin, which does everything anti-vendom would do - and more. Instead - I'd change the recipe to include Xander, give anti-venom the ability to rapidly remove reagents associated with wasteland animals, cazador venom, etc. For extra special bonus points, create an actual reagent for rad-scorpion stings, make their venom gland an item, and include the venom gland in the anti-venom recipe.

**Comment by SwiGs1** [comment]
I guess later down the line I could sprite some venom glands for mobs that use venom or poison in their attacks, I checked the mob code found it funny while Cazadores attack with their own reagent, radscorpions just shoot "toxin" I will be probably doing those changes when I work on some of the mob code when I have the time. Personally I was originally going to have Xander be an ingredient, but I just wanted to change it up with barrel cactus and then having to go out in caved areas for mushrooms, kinda reminded me of that quest in Jacobstown where you had to go to cave and Nightstalker cave and get blood and venom glands for the Doctor. But I just wanted a more tribal option, that's mostly what it is because you'll always see Legion carrying Antivenom instead of grinding up Xander Root for example. But overall I really like the idea.

**Comment by SwiGs1** [comment]
Screenshot (749)
Screenshot (750)
Screenshot (748)

Well, now its updated, Radscorpions and Cazadores now have their own venom reagents, they drop venom glands when butchered and venom glands now are used in the recipe to make antivenom and, antivenom now has a focus of removing those same wasteland animal venom reagents and stuff.

**Comment by imw-Oculus** [comment]
Fantastic! You've been wonderful, just make sure to tweak out any recipes that require carpotoxin, and replace them with cazador venom!

**Comment by SwiGs1** [comment]
@imw-Oculus What?

**Comment by SwiGs1** [comment]
Why would the cazador venom reagent be in a recipe it's the reagent that they inject you with that causes toxin damage, the glands they drop when you butcher them are used to craft anti venom

**Comment by imw-Oculus** [comment]
> Why would the cazador venom reagent be in a recipe it's the reagent that they inject you with that causes toxin damage, the glands they drop when you butcher them are used to craft anti venom, you should check the code, or I'm misunderstanding

Oh - lad, I just meant that there are a couple of medicine/toxin reagents that relies on carpotoxin, that's all.

**Pull request merged**